playbooks/common/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,18 @@
+---
+- include: ../openshift-cluster/evaluate_groups.yml
+
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: List known health checks
+    action: openshift_health_check
+    when: openshift_checks is undefined or not openshift_checks
```

Comment:
`pre_tasks` always run before `roles`.  Did you mean to use `post_tasks` here?  The order of execution for play stages is:
```yaml
- hosts: foo
  vars:
  pre_tasks:
  roles:
  tasks:
  post_tasks:
```
Putting these items in a different order in the yaml file does not change execution order.  Same with `vars:`, they are defined at the beginning of the play.